Index: work/src/org/kuali/kfs/fp/document/web/struts/ProcurementCardForm.java
===================================================================
--- work/src/org/kuali/kfs/fp/document/web/struts/ProcurementCardForm.java	(revision 5209)
+++ work/src/org/kuali/kfs/fp/document/web/struts/ProcurementCardForm.java	(revision 5211)
@@ -47,6 +47,8 @@
 
 import com.rsmart.kuali.kfs.fp.document.ProcurementCardDocument;
 
+import edu.arizona.kfs.sys.AZKFSConstants;
+
 /**
  * This class is the form class for the ProcurementCard document. This method extends the parent KualiTransactionalDocumentFormBase
  * class which contains all of the common form methods and form attributes needed by the Procurment Card document.
@@ -204,4 +206,13 @@
         // we'll just cheat...
         return "targetAccountingLines";
     }
+    
+    /**
+     * Customization for UA Sales Tax Amount and Tax Exempt Indicator.
+     * This method returns the value of the ENABLE_SALES_TAX_AMOUNT_TAX_EXEMPT_IND system parameter.
+     * @return
+     */
+    public boolean getEnableSalesTaxIndicator() {        
+        return SpringContext.getBean(ParameterService.class).getIndicatorParameter(ProcurementCardDocument.class, AZKFSConstants.ENABLE_SALES_TAX_AMOUNT_TAX_EXEMPT_IND);
+    }
 }
Index: work/web-root/WEB-INF/tags/fp/procurementCardTransactions.tag
===================================================================
--- work/web-root/WEB-INF/tags/fp/procurementCardTransactions.tag	(revision 5209)
+++ work/web-root/WEB-INF/tags/fp/procurementCardTransactions.tag	(revision 5211)
@@ -90,6 +90,19 @@
 		  </c:choose>
           </div></th>
        </tr>   
+       
+	   <tr>
+	      <c:if test="${KualiForm.enableSalesTaxIndicator}">
+	           <th> <div align="right"><kul:htmlAttributeLabel attributeEntry="${transactionAttributes.transactionSalesTaxAmount}"/></div></th>
+	           <td valign=top><kul:htmlControlAttribute attributeEntry="${transactionAttributes.transactionSalesTaxAmount}" property="document.transactionEntries[${ctr}].transactionSalesTaxAmount" readOnly="false"/></td>
+	           <th> <div align="right"><kul:htmlAttributeLabel attributeEntry="${transactionAttributes.transactionTaxExemptIndicator}"/></div></th>
+	           <td valign=top><kul:htmlControlAttribute attributeEntry="${transactionAttributes.transactionTaxExemptIndicator}" property="document.transactionEntries[${ctr}].transactionTaxExemptIndicator" readOnly="false"/></td>
+	      </c:if>
+	      <c:if test="${!KualiForm.enableSalesTaxIndicator}">
+	           <html:hidden write="false" property="document.transactionEntries[${ctr}].transactionSalesTaxAmount"/>
+	           <html:hidden write="false" property="document.transactionEntries[${ctr}].transactionTaxExemptIndicator"/>
+	      </c:if>
+	   </tr>       
     </table>   
 	
     <table width="100%" border="0" cellpadding="0" cellspacing="0" class="datatable">
